File: Obligatorio.API/Models/CarneSalud.cs
namespace Obligatorio.API;

public class CarneSalud
{
    public string ci { get; set; }
    public DateOnly fechaEmision { get; set; }
    public DateOnly fechavencimiento { get; set; }
    public string comprobante { get; set; }

    public CarneSalud(string _ci,
                      DateOnly _fechaEmision, 
                      DateOnly _fechavencimiento,
                      string _comprobante) 
    {
        this.ci = _ci;
        this.fechaEmision = _fechaEmision;
        this.fechavencimiento = _fechavencimiento;
        this.comprobante = _comprobante;
    }
}

File: Obligatorio.API/models/CarneSalud.ts
export class CarneSalud {

    public ci: String;
    public fechaEmision: Date;
    public fechaVencimiento: Date;
    public comprobante: string;

    constructor(
        _ci: String,
        _fechaEmision: Date,
        _fechaVencimiento: Date,
        _comprobante: string
    ) {
        this.ci = _ci
        this.fechaEmision = _fechaEmision
        this.fechaVencimiento = _fechaVencimiento
        this.comprobante = _comprobante
    }

}
